packages/uswds-core/src/styles/mixins/general/font-face.scss

```diff
@@ -17,9 +19,7 @@
     font-style: string.unquote($font-style);
     font-weight: $font-weight;
     font-display: fallback;
-    src: url(#{$file-path}.woff2) format("woff2"),
-      url(#{$file-path}.woff) format("woff"),
-      url(#{$file-path}.ttf) format("truetype");
+    src: font-sources($file-path);
```

Comment:
I've left this as a function since even though the logic is simple now, there may be some more complex logic we'll want to explore around variable fonts when we add that support in the future